src/pages/academy/grading/subcomponents/GradingSummary.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Badge,
+  Block,
+  Bold,
+  Flex,
+  List,
+  ListItem,
+  Metric,
+  ProgressBar,
+  Text,
+  Title
+} from '@tremor/react';
+import { AssessmentOverview, GradingStatuses } from 'src/commons/assessment/AssessmentTypes';
+import { GradingOverview } from 'src/features/grading/GradingTypes';
+
+import { AssessmentTypeBadge } from './GradingBadges';
+
+type GradingSummaryProps = {
+  group: string | null;
+  submissions: GradingOverview[];
+  assessments: AssessmentOverview[];
+};
+
+type AssessmentSummary = {
+  id: number;
+  type: string;
+  title: string;
+};
+
+const GradingSummary: React.FC<GradingSummaryProps> = ({ group, submissions, assessments }) => {
+  submissions = submissions.filter(({ assessmentType }) => assessmentType !== 'Paths');
+  const groupSubmissions = submissions.filter(
+    ({ groupName }) => group === null || groupName === group
+  );
+  const ungraded = groupSubmissions.filter(
+    ({ gradingStatus }) => gradingStatus !== GradingStatuses.graded
+  );
+  const ungradedAssessments = [...new Set(ungraded.map(({ assessmentId }) => assessmentId))].reduce(
+    (acc: AssessmentSummary[], assessmentId) => {
+      const assessment = assessments.find(assessment => assessment.id === assessmentId);
+      if (!assessment || assessment.type === 'Path') return acc;
+      return [
+        ...acc,
+        {
+          id: assessmentId,
+          type: assessment.type,
+          title: assessment.title
```

Comment:
<img width="2539" alt="Screenshot 2023-04-17 at 10 32 15 PM" src="https://user-images.githubusercontent.com/47914880/232518065-7156138a-cba3-4fc7-887b-e8676d0e03d3.png">

i realised that `AssessmentSummary::title` and `GradingOveriew::assessmentName` is mismatched in the test database, so it would be good to do a sanity check if the two attributes always match in prod db.

Reply:
perhaps in the future there could be a feature where users can click on a entry (the mission name) under "Ungraded assessments" and that will trigger a filtered view of grading dashboard showing assessments of that name (bonus: sorted by grading status)

Reply:
> <img alt="Screenshot 2023-04-17 at 10 32 15 PM" width="2000" src="https://user-images.githubusercontent.com/47914880/232518065-7156138a-cba3-4fc7-887b-e8676d0e03d3.png">
> 
> i realised that `AssessmentSummary::title` and `GradingOveriew::assessmentName` is mismatched in our test database, so it would be good to do a sanity check if the two attributes always match in prod db.

Should we do this check in the frontend? Ideally the assessment and grading data from the prod database should always match.

Reply:
> Ideally the assessment and grading data from the prod database should always match.

Then in that case, I would think it makes more sense if this check is done in the backend. The frontend would work on the assumption that this already matches.

@YaleChen299 thoughts on this?

Reply:
@RichDom2185 taking a another look, i realised the mismatch is likely caused by the mocks (see below) in our frontend and not our local dev database so it should be fine on staging/prod.

https://github.com/source-academy/frontend/blob/f2b05b2db3aee081f3d64f62926918b5c9f9f53d/src/commons/mocks/GradingMocks.ts#L10-L13

https://github.com/source-academy/frontend/blob/f2b05b2db3aee081f3d64f62926918b5c9f9f53d/src/commons/mocks/AssessmentMocks.ts#L90-L103

will raise a issue to fix and align our mocks in frontend